packages/webapp/src/views/FocusAreaSummary/schema.ts

```diff
@@ -0,0 +1,164 @@
+import faker from 'faker';
```

Comment:
mock data generation in production namespaces is a very big warning sign

While the schemas are only used in test code, let's move all schemas to the test namespaces.

When we want to use types in production code, we can pull the types out to the prod namespace and import them form the test code